.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+C:37:"PHPUnit\Runner\DefaultTestResultCache":665:{a:2:{s:7:"defects";a:0:{}s:5:"times";a:6:{s:93:"Tests\LoyaltyCorp\Multitenancy\Integration\Database\Entities\HasProviderTest::testHasProvider";d:0.047;s:112:"Tests\LoyaltyCorp\Multitenancy\Integration\Database\Entities\HasProviderTest::testHasProviderCanSaveNullProvider";d:0.004;s:90:"Tests\LoyaltyCorp\Multitenancy\Integration\Database\Entities\ProviderTest::testPersistence";d:0.003;s:77:"Tests\LoyaltyCorp\Multitenancy\Unit\Database\Entities\ProviderTest::testState";d:0;s:79:"Tests\LoyaltyCorp\Multitenancy\Unit\Database\Entities\ProviderTest::testNameSet";d:0;s:81:"Tests\LoyaltyCorp\Multitenancy\Unit\Database\Seeders\ProviderSeederTest::testSeed";d:0.004;}}}
```

Comment:
what is this file for?

Reply:
Weird. Looks like phpunit now outputs a cache?

Reply:
might be a good idea to add to ignore?